### ISSUE TITLE
CSP-EC Agreement (fill & sign)

### ISSUE THREAD

**Issue by dhachuel** [cornell-tech-qualcomm-challenge/admin#1]
Hi all, we need to fill, sign and send the CSP-EC agreements to Anand for him to sign. I suggest we send all 5 signed agreements with our first email so we get this out of the way. 

Thoughs?

David

**Comment by ZivJia** [comment]
I think we can sen him these agreements by email, right? 
Let's just put our own agreement on github
and then email him

**Comment by dhachuel** [comment]
Yes, I think we should do that. He has to sign them so make sure yours is signed before sending.
